optaplanner-build-parent/pom.xml

```diff
@@ -86,53 +86,13 @@
         <type>pom</type>
         <scope>import</scope>
       </dependency>
-      <!-- TODO use org.kie.kogito:kogito-bom instead once Kogito fixes https://issues.redhat.com/browse/KOGITO-2726 -->
       <dependency>
         <groupId>org.kie.kogito</groupId>
-        <artifactId>kogito-api</artifactId>
-        <version>${version.org.kie.kogito}</version>
-      </dependency>
-      <dependency>
-        <groupId>org.kie.kogito</groupId>
-        <artifactId>kogito-internal</artifactId>
-        <version>${version.org.kie.kogito}</version>
-      </dependency>
-      <dependency>
-        <groupId>org.kie.kogito</groupId>
-        <artifactId>kogito-drools-model</artifactId>
-        <version>${version.org.kie.kogito}</version>
-      </dependency>
-      <dependency>
-        <groupId>org.kie.kogito</groupId>
-        <artifactId>kogito-ruleunits</artifactId>
-        <version>${version.org.kie.kogito}</version>
-      </dependency>
-      <dependency>
-        <groupId>org.kie.kogito</groupId>
-        <artifactId>drools-core</artifactId>
-        <version>${version.org.kie.kogito}</version>
-      </dependency>
-      <dependency>
-        <groupId>org.kie.kogito</groupId>
-        <artifactId>drools-compiler</artifactId>
-        <version>${version.org.kie.kogito}</version>
-      </dependency>
-      <dependency>
-        <groupId>org.kie.kogito</groupId>
-        <artifactId>drools-core-dynamic</artifactId>
-        <version>${version.org.kie.kogito}</version>
-      </dependency>
-      <dependency>
-        <groupId>org.kie.kogito</groupId>
-        <artifactId>drools-core-static</artifactId>
-        <version>${version.org.kie.kogito}</version>
-      </dependency>
-      <dependency>
-        <groupId>org.kie.kogito</groupId>
-        <artifactId>drools-decisiontables</artifactId>
+        <artifactId>kogito-bom</artifactId>
```

Comment:
IIRC the kogito-bom is not a user bom. It's a platform bom, containing non org.kie.kogito artifacts.
If so, a big veto to import it.

Reply:
I've raised a conversation about this on the kogito chat some time ago. I am hoping they'll turn it into a user bom.

Reply:
FYI https://issues.redhat.com/browse/KOGITO-2726 is fixed.

Reply:
@ge0ffrey as @yurloc pointed out, since the version 0.14.0, the kogito-bom no longer contains other artifacts that org.kie.kogito, which is why I replaced the individual version declarations.